src/Assessor/AssessorRequestCommand.php

```diff
@@ -48,6 +48,7 @@ class AssessorRequestCommand
     private $firstName;
 
     /**
+     * @Assert\NotBlank(message="assessor.birth_name.not_blank", groups={"fill_personal_info"})
```

Comment:
The message `Votre nom de naissance est requis.` is generic, can you use instead : `common.birth_name.not_blank` ?

Like this new PR : https://github.com/EnMarche/en-marche.fr/pull/3610/files#diff-8f79de8cc8dbed0031a212b6338a458eR26

Reply:
Also, can you add the assert on the entity ?

Reply:
done, thanks :ok_hand: